framework/pym/play/commands/junit.py

```diff
@@ -38,16 +39,13 @@ def handle_sigint(signum, frame):
             process.kill()
 
 def execute(**kargs):
-    command = kargs.get("command")
     app = kargs.get("app")
     args = kargs.get("args")
-    env = kargs.get("env")
-    cmdloader = kargs.get("cmdloader")
-
     junit(app, args)
 
 def junit(app, args):
     app.check()
+    print("~ Running junit")
```

Comment:
is this needed?

Reply:
Not really, it rather was confirmation of what's executed on CI to tell apart from https://github.com/Nosto/play1/blob/master/framework/pym/play/commands/test.py#L25 , i don't mind cleaning this up